paws/templates/legacy.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    name: paws-public-legacy
+    ingress.paws.wmcloud.org: legacy
+  annotations:
+    nginx.ingress.kubernetes.io/temporal-redirect: $scheme://{{ .Values.pawspublic.ingress.host }}/$2
```

Comment:
This will drop query strings from the redirect. Not sure if that has any effect here. Adding `$is_args$args` at the end of the redirect will preserve query string data.

Reply:
Query strings on paws public are possibly usable? I hadn't considered that likely or possible, but you might be right that people might use it as a generic web server or something.

Reply:
Done 3bbe5ee  
I livehacked it in to the running cluster to see how it behaved, and it seemed fine.